lib/std/zig/render.zig

```diff
@@ -1926,13 +1926,15 @@ fn renderContainerDecl(
 
     const src_has_trailing_comma = token_tags[rbrace - 1] == .comma;
     if (!src_has_trailing_comma) one_line: {
-        // We print all the members in-line unless one of the following conditions are true:
 
         // 1. The container has comments or multiline strings.
         if (hasComment(tree, lbrace, rbrace) or hasMultilineString(tree, lbrace, rbrace)) {
             break :one_line;
         }
 
+        // 2. The container has a container comment.
```

Comment:
Nit: using `// 2.` for the comment here means that the following comments are off-by one and should be updated.